pkg/jobrunaggregator/jobrunaggregatorlib/disruption.go

```diff
@@ -0,0 +1,74 @@
+package jobrunaggregatorlib
+
+import (
+	"fmt"
+	"math"
+	"regexp"
+	"strings"
+	"time"
+)
+
+var (
+	UpgradeBackendNameToTestSubstring = map[string]string{
```

Comment:
straight move?

Reply:
Yea direct move to lib, since it's being used in the analyzer now too